components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -62,6 +70,10 @@ export class ProjectsDropdownComponent implements OnInit, OnChanges {
     if (this.disabled) {
       return;
     }
+    if (!this.active) {
+      this.filterValue = '';
+      this.filteredProjects = this.projectsArray;
+    }
```

Comment:
only clear the filterValue out and reset the filtered projects list when we're done with the dropdown